backend/__tests__/routes/admins.test.js

```diff
@@ -85,7 +84,7 @@ describe("PATCH: Update user role", () => {
             },
             {
                 where: {
-                    AccountId: "51",
+                    AccountId: "828",
```

Comment:
should we move this id to a constant? I see that you use it again lower

Reply:
great idea !